perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -367,6 +371,16 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
   confidence = ('-I 99,5 -i {0},3'.format(FLAGS.netperf_max_iter)
                 if FLAGS.netperf_max_iter else '')
   verbosity = '-v2 ' if enable_latency_histograms else ''
+
+  netperf_cmd = ""
+
+  remote_cmd_timeout = \
```

Comment:
nit: Avoid backslash for line continuation. Use parentheses instead:

remote_cmd_timeout = (
    FLAGS.netperf_test_length * (FLAGS.netperf_max_iter or 1) + 300)

Reply:
Changed to parentheses